std_msgs/Header header
std_msgs/Int16 marker_dictionary
std_msgs/Int16[] marker_ids
geometry_msgs/PoseStamped[] marker_poses